From e3cf64310ffdce3b3623ee522dd60ec9689b75cf Mon Sep 17 00:00:00 2001
From: Heinrich Schuchardt <xypron.glpk@gmx.de>
Date: Sat, 25 Aug 2018 07:45:37 +0200
Subject: [PATCH 1/1] [build] Don't use -mabi=lp64 for HOSTCC

Patch 1c47eb186e7e ("[build] Conditionalise use of -mabi=lp64 for ARM64
builds") causes HOSTCC util/elf2efi64 to be called with -mabi=lp64
which is illegal on x86_64.

Fixed: 1c47eb186e7e ("[build] Conditionalise use of -mabi=lp64 for ARM64
builds")
Signed-off-by: Heinrich Schuchardt <xypron.glpk@gmx.de>

Signed-off-by: Heinrich Schuchardt <xypron.glpk@gmx.de>
---
 src/arch/arm64/Makefile | 1 -
 1 file changed, 1 deletion(-)

diff --git a/src/arch/arm64/Makefile b/src/arch/arm64/Makefile
index 9b9dd5ec..388dcd32 100644
--- a/src/arch/arm64/Makefile
+++ b/src/arch/arm64/Makefile
@@ -16,7 +16,6 @@ ASFLAGS		+= -mabi=lp64 -EL
 ifeq ($(CCTYPE),gcc)
 LP64_TEST = $(CC) -mabi=lp64 -x c -c /dev/null -o /dev/null >/dev/null 2>&1
 LP64_FLAGS := $(shell $(LP64_TEST) && $(ECHO) '-mabi=lp64')
-WORKAROUND_CFLAGS += $(LP64_FLAGS)
 endif
 
 # EFI requires -fshort-wchar, and nothing else currently uses wchar_t
-- 
2.18.0

